.github/workflows/continuous-integration.yaml

```diff
@@ -0,0 +1,92 @@
+name: 'Continuous integration'
+on: ['push', 'pull_request']
+jobs:
+  cs:
+    runs-on: 'ubuntu-20.04'
+    name: 'Coding style'
+    steps:
+      - name: 'Checkout'
+        uses: 'actions/checkout@v2'
+
+      - name: 'Setup PHP'
+        uses: 'shivammathur/setup-php@v2'
+        with:
+          php-version: '7.2'
+          coverage: 'none'
+          extensions: 'json, mbstring, tokenizer'
+          tools: 'composer-normalize:2.13.2, php-cs-fixer:2.18.2'
+
+      - name: 'Check PHP code'
+        run: |
+          php-cs-fixer fix --diff --dry-run --allow-risky=yes --using-cache=false
+
+      - name: 'Check composer.json'
+        run: |
+          composer-normalize --diff --dry-run --indent-size=4 --indent-style=space --no-update-lock
```

Comment:
I have the suspicion that this will return a lot of errors as we tweaked the rules over time. We can either fix them or loosen the rules.